tests/test_metrics.py

```
@@ -0,0 +1,56 @@
+        assert result["CharCount"] == {"number": 5}
+
+    def test_ticker_count_case_insensitive(self):
+        result = extract_briefing_metrics("PLTR は上昇、nvda は横ばい", tickers=["PLTR", "NVDA"])
+        assert result["TickerCount"] == {"number": 2}
+
+    def test_ticker_not_mentioned(self):
+        result = extract_briefing_metrics("市場は安定", tickers=["PLTR", "NVDA"])
+        assert result["TickerCount"] == {"number": 0}
+
+    def test_partial_match_not_counted(self):
+        # "NVDA" を含む単語 "NVDA-related" でも部分一致はカウントされる（IN 検索）
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)